src/main/java/org/littil/api/module/repository/ModuleService.java

```diff
@@ -0,0 +1,23 @@
+package org.littil.api.module.repository;
+
```

Comment:
This service class is in the module.repository package instead of module.service.

Reply:
moved to service package